cmd/metadata/service/token.go

```diff
@@ -167,6 +170,35 @@ func (s *TokenService) saver(ctx context.Context) {
 				continue
 			}
 
+			for i := range tokens {
```

Comment:
Looks like you can make a function here to avoid duplicated code. It's a copy-pasted part from another case section.

Reply:
done